docker-load.md

```diff
@@ -60,28 +60,27 @@ _Note that the created field may not match._
 
 #### Next steps for Windows 10 Pro users
 
-Make sure you have 7-Zip installed.
-If you do not, you can download the `64-bit x64` program here: https://www.7-zip.org/
+2. Install 7-Zip install if you do not already have it by downloading the `64-bit x64` program here: https://www.7-zip.org/
```

Comment:
Would the version of 7-Zip installed (19.00 vs 19.02 alpha) matter in this case? If so, this should also be specified here.

Reply:
I don't know but it seems highly unlikely.